targetcli/ui_backstore.py

```diff
@@ -785,3 +784,15 @@ def summary(self):
             config_str = so.config[idx+1:]
 
         return ("%s (%s) %s" % (config_str, bytes_to_human(so.size), so.status), True)
+
+    def ui_command_changemedium(self, size, cfgstring):
+        size = human_to_bytes(size)
+        config = self.handler + "/" + cfgstring
```

Comment:
It seems to crash here. handler is set on the UIUserBackedBackstore and not set on the UIUserBackedStorageObject. I fixed up UIUserBackedStorageObject but then flip flopped and just put it on the UIUserBackedBackstore like you ad it originally since both create/remove are there.

I fixed some other items in the patch:
1. pass in device name to runner in self.iface.ChangeMedium()
2. Fix up error messages.

[targetcli-change-medium.txt](https://github.com/open-iscsi/targetcli-fb/files/2300654/targetcli-change-medium.txt)